products/BellHybrid/apps/common/include/common/windows/BellFinishedWindow.hpp

```diff
@@ -28,8 +28,8 @@ namespace gui
         void onBeforeShow(ShowMode mode, SwitchData *data) override;
 
         Icon *icon{};
-        std::string windowToReturn;
-        bool closeApplication; // is close application is true windowToReturn variable is ignored
+        std::string windowToReturn{};
```

Comment:
There is no need to explicitly brace initialize it, as without it `std::string` default constructor will be invoked.
```suggestion
        std::string windowToReturn;
```